src/fc_checks.c

```diff
@@ -232,3 +232,181 @@ struct check_result *check_file_context_types_exist(__attribute__((unused)) cons
 
 	return NULL;
 }
+
+/**********************************
+ * Return true if the two fc_entry nodes are the same
+ * and false otherwise
+ **********************************/
+static bool is_same_fc_entry(const struct fc_entry *entry_one,
+		const struct fc_entry *entry_two) {
+
+	return !strcmp(entry_one->path, entry_two->path)
```

Comment:
This is a tone of indentation.  I get that you're trying to make the complicated block clear, but can you cut back to one tab at each level instead of two?